templates/bigquery-telemetry.jinja

```diff
@@ -25,6 +25,9 @@ resources:
       tableId: {{ TABLE_NAME }}
     schema:
           fields:
+          - name: vehicle
+            type: string
+            mode: REQUIRED
```

Comment:
yep actually, I got an error when I tried to update so I went and deleted the deployment, then rebuilt it.  I didn't look too deep into how this could be set up in the deployment manager, but figured that we'd hopefully just stick with the new data table format so we don't have to change it again!

Reply:
Good to know!

I doubt we'll be changing the schema often so it's not a huge deal; it doesn't really look like there's an easy way to do this with Deployment Manager anyway unless we write a script template that manually applies the update migrations. Definitely not worth it for something that's rather inconsequential.